docs/usage/models.md

```diff
@@ -738,7 +738,7 @@ Also, like `List` and `Dict`, any parameters specified using a `TypeVar` can lat
     `TypeVar` needs to be wrapped inside [`SerializeAsAny`](serialization.md#serializing-with-duck-typing)
     for Pydantic to serialize `ChildModel` as `ChildModel`.
 
-```py
+```py requires="3.12"
```

Comment:
Is it only for tests? Would it still show different example code for different Python versions?

Reply:
No, it shows the same thing for every Python version.